src/Sarif.UnitTests/Writers/SarifLoggerTests.cs

```diff
@@ -34,24 +33,38 @@ public void SarifLogger_RedactedCommandLine()
             // "C:\Program Files (x86\\Microsoft Visual Studio 14.0\Common7\IDE\QTAgent32_40.exe\" 
             // /agentKey a144e450-ac06-46d0-8365-c21ea7872d23 /hostProcessId 8024 /hostIpcPortName 
             // eqt -60284c64-6bc1-3ecc-fb5f-a484bb1a2475"
+            // 
```

Comment:
It took me a while to figure out why this was needed, and I think a comment is in order, something like this:

> On a developer's machine, the script BuildAndTest.cmd runs the tests with a particular command line. Under AppVeyor, the appveyor.yml file simply specifies the names of the test assemblies, and AppVeyor constructs and executes its own, different command line. So, based on our knowledge of each of those command lines, we select a different token to redact in each of those cases.